charts/shoot-cloud-config/charts/original/templates/kubelet/_kubelet-config-file

```diff
@@ -53,8 +53,11 @@ evictionPressureTransitionPeriod: 4m0s
 evictionMaxPodGracePeriod: 90
 evictionMinimumReclaim: null
 failSwapOn: true
-{{- if .kubernetes.kubelet.featureGates }}
 featureGates:
+{{- if semverCompare "< 1.11" .kubernetes.version }}
```

Comment:
Add this logic also in 
https://github.com/gardener/gardener/blob/92ca7287cf2cb7287410c6130fae4b32c9201dd4/charts/shoot-cloud-config/charts/original/templates/kubelet/_kubelet.flags#L25-L27

Reply:
Done. Refer - https://github.com/gardener/gardener/pull/267/commits/062e57bda20c41dce228a5c4159f28b5eb766de3?utf8=%E2%9C%93&diff=unified#diff-7a590da443a0105c0f781f2e0fd2c726R28